examples/ner/run_pl.sh

```diff
@@ -1,6 +1,20 @@
-# Require pytorch-lightning=0.6
+# Install newest ptl.
+!pip install -U git+http://github.com/PyTorchLightning/pytorch-lightning/
```

Comment:
is this `!` at the start supposed to be here?  or an artifact of porting from jupyter?